src/styles/shared-styles.scss

```diff
@@ -29,21 +29,27 @@
 }
 
 .ms-Icon--ChevronDown::before {
-  content: "\\\E70D";
+  content: '\\\E70D';
 }
 
 .ms-Icon--ChevronUp::before {
-  content: "\\\E70E";
+  content: '\\\E70E';
 }
 
 .ms-Icon--Contact::before {
-  content: "\\\E77B";
+  content: '\\\E77B';
 }
 
 .ms-Icon--AddFriend::before {
-  content: "\\\E8FA";
+  content: '\\\E8FA';
 }
 
 .ms-Icon--OutlookLogoInverse::before {
-  content: "\\\EB6D";
+  content: '\\\EB6D';
 }
+
+$presence-color-online: #6bb700;
```

Comment:
I don't see these used outside of the person component, they should stay with the person component.